lib/jeckle/rest_actions.rb

```diff
@@ -13,11 +13,11 @@ def find(id)
       end
 
       def search(params = {})
-        response   = run_request(resource_name, params).response.body || []
+        response   = run_request(params.delete[:resource_name] || resource_name, params).response.body || []
```

Comment:
@brennovich before make some opinions about it I would like to hear @cairesr cases. @cairesr can you give some **fake** GET/POST examples about your different scenario? I ask because if we accept this we need to support in all the restful actions that will be added/ More description of this feature that will be implemented on #32.